backend/internal/websocket/manager.go

```diff
@@ -39,7 +38,10 @@ func NewClientManager(usecases *usecase.Usecases) *ClientManager {
 // Run starts the manager to handle register/unregister clients and broadcasting messages
 func (manager *ClientManager) Run(ctx context.Context) {
 	// Start the periodic message sender and query the db via usecases
-	go manager.sendTotalExecutedTXs(ctx, 5*time.Second, "included")
+	go manager.sendTotalExecutedTransactions(ctx, 60*time.Second, "included")
```

Comment:
What's the point of running `manager.send...` in its own go routine, since it only spawns a new go routine?

Reply:
yeah, since internally its already firing a routine we don't need to run the handlers in a separate routine.